code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm

```diff
@@ -316,6 +392,146 @@
 	word1 = /datum/cultword/hell
 	word2 = /datum/cultword/technology
 	word3 = /datum/cultword/join
+	cost_invoke = 2
+	cost_upkeep = 1
+	remaining_cost = 5
+	talisman_absorb = RUNE_CANNOT
+	var/obj/item/weapon/tome/target = null
+	var/obj/item/weapon/talisman/tool = null
+	page = "Runes are powerful, but they're not always convenient. They require time to be set up, cannot be moved, and more importantly, are highly visible, unless hidden, \
+		which would require additional preparation. With that in mind, cultists need an alternative to use their powers reliably. This rune provides that alternative in the form \
+		of talismans. Created in exchange of a drop of blood, these sheets can absorb a rune, and then be used to channel its power. They're easy to conceal until used, and can be stored \
+		inside an arcane tome however most talismans are weaker than the rune they're imbued from. Exceptions exist, as the Stun rune which becomes much more potent when used directly in contact with a target. \
+		Other exceptions are the Door, Portal Entrance, and Conversion runes which will be attuned with the talisman instead of absorbed, allowing them to be triggered remotely, and also \
+		the Portal Exit rune, which can be used to immediately jaunt toward the Exit it was attuned to. One last use for this rune, is a ritual allowing a talisman to be transmitted directly \
+		inside an arcane tome carried by a fellow cultist. The ritual takes a bit of time and blood, but can save your acolyte some precious time."
+
+
+/datum/rune_spell/conjuretalisman/cast()
+	var/obj/item/weapon/talisman/AT = locate() in get_turf(spell_holder)
+	if (AT)
+		if (AT.spell_type)
+			var/mob/living/user = activator
+			var/list/valid_tomes = list()
+			var/i = 0
+			for (var/obj/item/weapon/tome/T in arcane_tomes)
+				i++
+				var/mob/M = T.loc
+				if (!M)	continue
+				if (!istype(M))
+					M = M.loc
+				if (!istype(M))
+					M = M.loc
+				if (!istype(M))
+					continue
+				else
+					valid_tomes.Add("[i] - Tome carried by [M.real_name] ([T.talismans.len]/[MAX_TALISMAN_PER_TOME])")
```

Comment:
you can remove this line and all will be the same

Reply:
I've been told that before but for some raison I feel paranoid about it.